shared-helpers/src/regions.ts

```diff
@@ -0,0 +1,59 @@
+export enum Region {
```

Comment:
Just moved this file out of ui-components, the diff didn't follow

Reply:
Thank you for these comments, they're helpful!